### PR TITLE
Adding PNM API support.

### PR DESCRIPTION
**Pull request by goru97**
# What:
Adding API support to CRUD private monitoring zones.

### CODE REVIEW

File: commands/raxmon-monitoring-zones-update [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import optparse
+
+from raxmon_cli.common import run_action
+from raxmon_cli.utils import instance_to_dict
+
+OPTIONS = [
+    [['--id'], {'dest': 'id', 'help': 'Entity id'}],
```
Comment (rjemanuele) [inline commands/raxmon-monitoring-zones-update]: Shouldn't that be zone id?

File: commands/raxmon-monitoring-zones-create [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from raxmon_cli.common import run_action
+
+OPTIONS = [
+    [['--label'], {'dest': 'label', 'help': 'Entity label'}],
```
Comment (rjemanuele) [inline commands/raxmon-monitoring-zones-create]: Shouldn't that be zone label?

File: commands/raxmon-monitoring-zones-update (hunk 1 above)
Comment (goru97) [inline commands/raxmon-monitoring-zones-update]: It's the same for all the other objects (like entities, checks etc). Just following the convention.

File: commands/raxmon-monitoring-zones-create (hunk 2 above)
Comment (goru97) [inline commands/raxmon-monitoring-zones-create]: likewise

Comment (goru97) [inline commands/raxmon-monitoring-zones-create]: I am sorry. totally blind

**Comment by rjemanuele** [comment]
@goru97 please update the version number, someone before may have forgotten to note the changes in the change log when the did bump the version in the __init__.py

**Comment by rjemanuele** [comment]
One more thing, this cli requires 0.7.1 of the library.
https://github.com/racker/rackspace-monitoring-cli/blob/master/setup.py#L92

**Pull request merged**